src/python/py_vlm_pipeline.cpp

```diff
@@ -88,8 +88,9 @@ ov::AnyMap vlm_kwargs_to_any_map(const py::kwargs& kwargs, bool allow_compile_pr
         } else if (key == "streamer") {
```

Comment:
as far as I can see all these params `images`, `generation_config`, `streamer` etc can now be passed to `VLMPipeline` ctor, which is the right thing. Those options are `generate` properties.

Reply:
> I suppose we also need to fix it for text2image?

yes